daemon/policy_test.go

```diff
@@ -972,3 +974,10 @@ func (ds *DaemonSuite) Test_addCiliumNetworkPolicyV2(c *C) {
 		c.Assert(args.repo.GetRulesList().Policy, checker.DeepEquals, want.repo.GetRulesList().Policy, Commentf("Test name: %q", tt.name))
 	}
 }
+
+type FakeCIDRIdentityManager struct{}
```

Comment:
Can this live in `pkg/testutils` and be shared with the other `FakeCIDRIdentityManager`?

Reply:
Done